src/shogun/kernel/Kernel.h

```diff
@@ -27,7 +27,6 @@ namespace shogun
 {
 	class CFile;
 	class CFeatures;
-	class CKernelNormalizer;
```

Comment:
@vigsterkr Could this be an issue?

Reply:
unlikely as we have `#include <shogun/kernel/normalizer/KernelNormalizer.h>` before the class :)

Reply:
OK! just finding all the kernelnormalizer that were changed...